config/src/test/java/org/axonframework/config/EventProcessingModuleTest.java

```diff
@@ -339,8 +344,8 @@ public void testConfigureDefaultErrorHandler() throws Exception {
     }
 
     @Test
-    public void testTrackingProcessorsUsesConfiguredDefaultStreamableMessageSource() {
-        StreamableMessageSource<TrackedEventMessage<?>> mock = mock(StreamableMessageSource.class);
+    void testTrackingProcessorsUsesConfiguredDefaultStreamableMessageSource(
+            @Mock StreamableMessageSource<TrackedEventMessage<?>> mock) {
```

Comment:
Curious: @AlexElin, do you feel this parameter annotation approach is cleaner for tests then the static `mock` call?

Reply:
This fixes Unchecked assignment warning. (https://javadoc.io/static/org.mockito/mockito-junit-jupiter/3.1.0/org/mockito/junit/jupiter/MockitoExtension.html)

Reply:
But of course, fair point @AlexElin!